types/blogHeader.ts

```diff
@@ -1,4 +1,10 @@
 export interface HeaderProps {
   logo: string | null;
   blogName: string;
+  navList: {
+    id: number;
+    name: string;
+    navUrl: string;
+    isPage: boolean;
+  }[];
```

Comment:
갠적으로는 
```suggestion
  navList: Array<{
    id: number;
    name: string;
    navUrl: string;
    isPage: boolean;
  }>;
```
를 더 좋아하지만,,,갠취니까! 참고스